src/dict/loaddictionaries.cc

```diff
@@ -228,8 +232,6 @@ void loadDictionaries( QWidget * parent,
     QMessageBox::critical( parent,
                            QCoreApplication::translate( "LoadDictionaries", "Error loading dictionaries" ),
                            QString::fromUtf8( loadDicts.getExceptionText().c_str() ) );
```

Comment:
Here should show all errors.

Reply:
Error handling should be religious, and silently ignoring errors is sinful 😅.

I think this only need to append text to a QStringList, then call QStringList::join('\n') here.

BTW, I think there should be a systematical way to report dict errors and warnings back to users.

- https://github.com/xiaoyifang/goldendict-ng/issues/1440

Reply:
This can be left from another PR.